_tutorials/convert-html-site-to-jekyll.md

```diff
@@ -0,0 +1,362 @@
+    {% endfor %}
+</ul>{% endraw %}
+```
+
+If you have more sophisticated requirements around navigation, see the [detailed tutorial on navigation]({% link _tutorials/navigation.md %}). 
+
+
+## 10. RSS feed
+
+Your Jekyll site needs an RSS feed. Here's the [basic RSS feed syntax](http://www.w3schools.com/xml/xml_rss.asp). To create an RSS file in Jekyll, create a file called `feed.xml` and add the following:
+
+```xml
+---
+layout: null
+---
+
+{% raw %}<?xml version="1.0" encoding="UTF-8" ?>
+<rss version="2.0">
+
+    <channel>
+        <title>{{ site.title }}</title>
+        <link>{{ site.url }}</link>
+        <description>{{ site.description }}</description>
+        <lastBuildDate>{{ site.time | date_to_rfc822 }}</lastBuildDate>
+        {% for post in site.posts %}
+        <item>
+            <description>
+                {{ post.content | escape | truncate: '400' }}
+            </description>
+            <pubDate>{{ post.date | date_to_rfc822 }}</pubDate>
+            <guid>
+                {{ post.url | prepend: site.baseurl | prepend: site.url }}
+            </guid>
+        </item>
+        {% endfor %}
+    </channel>
+</rss>{% endraw %}
+```
+
+Make sure your `_config.yml` file has properties for `title`, `url` (and potentially `baseurl`), and `description`.
+
+This code uses a `for` loop to look through your last 20 posts. The content from the posts gets escaped and truncated to the last 400 characters using [Liquid filters](https://help.shopify.com/themes/liquid/filters).
+
+In your `default.html` layout, look for a reference to the RSS or Atom feed in your header, and replace it with a reference to the file you just created.
+
+```html
+<link rel="alternate" type="application/rss+xml"  href="/feed.xml" title="{% raw %}{{ site.title }}{% endraw %}">
+```
+
+Note that if you're publishing your site on Github Pages, you can add a Github Pages gem called [`jekyll-feed`][jekyll-feed] to auto-generate your feed.
+
+## 11. Add a sitemap
+
+Finally, add a [site map](https://www.sitemaps.org/protocol.html) in your root directory. Create a `sitemap.xml` file and add this code:
+
+```xml
+---
+layout: null
+search: exclude
+---
+
+{% raw %}<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+
+    {% for page in site.pages %}
+    <url>
+        <loc>{{page.url}}</loc>
+        <lastmod>{{site.time | date: '%Y-%B-%d' }}</lastmod>
+        <changefreq>daily</changefreq>
+        <priority>0.5</priority>
+    </url>
+    {% endfor %}
+
+    {% for post in site.posts %}
+    <url>
+        <loc>{{post.url}}</loc>
+        <lastmod>{{site.time | date: '%Y-%B-%d' }}</lastmod>
+        <changefreq>daily</changefreq>
+        <priority>0.5</priority>
+    </url>
+    {% endfor %}
+   
+</urlset>{% endraw %}
+```
+
+Note that if you're publishing your site on Github Pages, you can add a Github Pages gem called [`jekyll-sitemap`][jekyll-sitemap] to auto-generate your sitemap.
```

Comment:
same here `jekyll-sitemap` does not require for you to use GitHub Pages. I think what you mean it's that this gem is whitelisted by GitHub Pages.